libs/wingsdk/src/http/http.ts

```diff
@@ -181,7 +181,11 @@ export class Util {
     url: string,
     options?: RequestOptions
   ): Promise<Response> {
-    const res = await fetch(url, { ...defaultOptions, ...options });
+    const hasProtocol = url.match(/^(.*):\/\/(.*)/);
```

Comment:
This feels like a bad practice. I will be more comfortable with just failing with a nice error if the URL doesn't have a protocol and documenting that the URL requires a protocol. Assuming `http://` could be unsafe and surprising.